bspump/lookup/memcachedlookup.py

```diff
@@ -0,0 +1,57 @@
+from ..abc.lookup import Lookup
+from pymemcache.client import base
+
+import logging
+
+L = logging.getLogger(__name__)
+
+###
+
+
+class Memcachedookup(Lookup):
+
+	ConfigDefaults = {
+		'max_size': 1000,
+		'expiry_seconds': 3600,
+		'memcache': '127.0.0.1:11211',
```

Comment:
`server`

... and also check if/how multiple addresses can be supplied.

Reply:
@ateska  there exists memcached cluster, should we use this? I pasted a snippet below how it works

Using a memcached cluster,

This will use a consistent hashing algorithm to choose which server to set/get the values from. It will also automatically rebalance depending on if a server goes down.

from pymemcache.client.hash import HashClient

client = HashClient([
    '127.0.0.1:11211',
    '127.0.0.1:11212',
])
client.set('some_key', 'some value')
result = client.get('some_key')

Reply:
Ok, let's discuss this internally first and we will implement that eventually in the other MR.